src/test/resources/org/rust/lang/core/parser/fixtures/complete/expr.rs

```diff
@@ -47,6 +47,7 @@ fn test() -> u32 {
     move |x: i32| {
         x
     };
+    const || {};
```

Comment:
BTW we don't have test for async closures
Also, it would be nice to have a closure expr with all possible keywords to commit the order